build_projects/dotnet-cli-build/DotNetNew.cs

```diff
@@ -12,16 +12,18 @@ protected override string Command
 
         protected override string Args
         {
-            get { return $"{GetTemplateType()}"; }
+            get { return $"{GetTemplateType()} {TemplateArgs}"; }
         }
 
         public string TemplateType { get; set; }
 
+        public string TemplateArgs { get; set; }
+
         private string GetTemplateType()
```

Comment:
No need in keeping this method around since it just returns `TemplateType` now.  Might as well make it consistent with `TemplateArgs`.